js_modules/dagit/packages/core/src/app/QueryRefresh.tsx

```diff
@@ -111,6 +118,35 @@ export function useQueryRefreshAtInterval(queryResult: QueryResult<any, any>, in
   );
 }
 
+/**
+ * This hook allows you to hook a single QueryRefreshCountdown component to more than
+ * one useQueryRefreshAtInterval. The QueryRefreshCountdown will reflect the countdown
+ * state of the FIRST query, but clicking the refresh button will trigger them all.
+ *
+ * Note: If you use this hook, you should pass the same interval to each
+ * useQueryRefreshAtInterval.
+ */
+export function useMergedRefresh(
+  a: QueryRefreshState,
+  b: QueryRefreshState,
+  c?: QueryRefreshState,
+  d?: QueryRefreshState,
```

Comment:
Decided not to make this an array because useMemo on :147 wanted specifically enumerated arguments. Could also make it an array and silence that warning though 😅

Reply:
Is four an arbitrary choice?